internal/start/start.go

```diff
@@ -94,6 +95,13 @@ func run(p utils.Program) error {
 		},
 	)
 
+	// matching against public interface because concrete type is private, error.As only works for types */
+	if _, ok := err.(errdefs.ErrConflict); ok {
```

Comment:
We should add a unit test for this code path to be sure. There's a few examples of [mocking docker api](https://github.com/supabase/cli/blob/main/internal/utils/docker_test.go) in tests.

Let me know if you have to time to do it or I can take a look.

Reply:
I'll give it a shot 👍 thanks for the example link